CONTRIBUTING.md

```diff
@@ -1,26 +1,101 @@
-# Dynatrace Operator
+- [Contributing](#contributing)
+  - [How to run unit tests](#how-to-run-unit-tests)
+  - [How to run E2E tests](#how-to-run-e2e-tests)
+  - [Coding style guide](#coding-style-guide)
+    - [General](#general)
```

Comment:
Missing item (code-review):
https://github.com/go-critic/go-critic/blob/master/CONTRIBUTING.md#code-review

Reply:
Done :+1: I added a link to it, left it on TODO: and I think we can extend it with time